packages/components/Button/src/FAB/FABCore.tsx

```diff
@@ -65,20 +66,54 @@ export const FAB = compose<FABType>({
       }
       const label = accessibilityLabel ?? childText;
 
-      const fabWithoutShadow = (
-        <Slots.root {...mergedProps} accessibilityLabel={label}>
+      const buttonContent = (
+        <React.Fragment>
           {icon && <Slots.icon {...iconProps} />}
-          {React.Children.map(children, (child) =>
-            typeof child === 'string' ? <Slots.content key="content">{child}</Slots.content> : child,
-          )}
-        </Slots.root>
+          {showChildren &&
```

Comment:
Is it possible to take out the showChildren logic out of this PR now that it's in another one?